raster/r.in.pdal/testsuite/test_r_in_pdal_extent.py

```diff
@@ -0,0 +1,171 @@
+        """Test extent is automatically reprojected from epsg 6346"""
+        tools = Tools()
+        extent = tools.r_in_pdal(input=self.point_file_6346, flags="g").text
+        extent = gs.parse_key_val(extent, vsep=" ", val_type=float)
+        self.assertAlmostEqual(extent["n"], 222012.882582, places=6)
+        self.assertAlmostEqual(extent["s"], 222000.743357, places=6)
+        self.assertAlmostEqual(extent["e"], 637511.895399, places=6)
+        self.assertAlmostEqual(extent["w"], 637499.697426, places=6)
+        self.assertAlmostEqual(extent["b"], 100, places=6)
+        self.assertAlmostEqual(extent["t"], 109, places=6)
+
+    @unittest.skipIf(shutil.which("r.in.pdal") is None, "Cannot find r.in.pdal")
+    def test_no_reprojection_needed_extent(self):
+        """Test extent matches input 3358 data"""
+        tools = Tools()
+        extent = tools.r_in_pdal(input=self.point_file_3358, flags="g").text
+        extent = gs.parse_key_val(extent, vsep=" ", val_type=float)
+        self.assertAlmostEqual(extent["n"], 222012, places=6)
+        self.assertAlmostEqual(extent["s"], 222000, places=6)
+        self.assertAlmostEqual(extent["e"], 637512, places=6)
+        self.assertAlmostEqual(extent["w"], 637500, places=6)
+        self.assertAlmostEqual(extent["b"], 100, places=6)
+        self.assertAlmostEqual(extent["t"], 109, places=6)
+
+    @unittest.skipIf(shutil.which("r.in.pdal") is None, "Cannot find r.in.pdal")
+    def test_override_extent(self):
+        """Test extent matches input 3358 data"""
+        tools = Tools()
+        extent = tools.r_in_pdal(input=self.point_file_3358, flags="go").text
+        extent = gs.parse_key_val(extent, vsep=" ", val_type=float)
+        self.assertAlmostEqual(extent["n"], 222012, places=6)
+        self.assertAlmostEqual(extent["s"], 222000, places=6)
+        self.assertAlmostEqual(extent["e"], 637512, places=6)
+        self.assertAlmostEqual(extent["w"], 637500, places=6)
+        self.assertAlmostEqual(extent["b"], 100, places=6)
+        self.assertAlmostEqual(extent["t"], 109, places=6)
+
+    @unittest.skipIf(shutil.which("r.in.pdal") is None, "Cannot find r.in.pdal")
+    def test_no_reprojection_needed_flags_e(self):
+        """Test 3358 data is imported correctly"""
+        tools = Tools()
+        tools.g_region(n=1, s=0, e=1, w=0, res=1)
+        tools.r_in_pdal(input=self.point_file_3358, output=self.output, flags="e")
+        self.assertRasterExists(self.output)
+        rinfo = tools.r_info(map=self.output, flags="gr", format="json")
+        self.assertAlmostEqual(222012, rinfo["north"])
+        self.assertAlmostEqual(222000, rinfo["south"])
+        self.assertAlmostEqual(637512, rinfo["east"])
+        self.assertAlmostEqual(637500, rinfo["west"])
+        # The extent shows min=100, that point is filtered out
+        # because it's at the edge, this may not be ideal behavior
+        self.assertAlmostEqual(101, rinfo["min"])
```

Comment:
I agree this is strange, esp. with the expected result below. I recall seeing something like that before probably with the original r.in.lidar code. I agree with leaving it as is for this PR.